src/test/kotlin/racingcar/application/RacingGameTest.kt

```diff
@@ -0,0 +1,28 @@
+package racingcar.application
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.inspectors.forAll
+import io.kotest.matchers.shouldBe
+import racingcar.application.Car.Companion.createDefault
+
+class RacingGameTest : StringSpec({
+    "canMove가 true일 경우 차들은 전진한다" {
+        val decider = CarMovementDecider { true }
+        val cars = listOf(createDefault("car1"), createDefault("car2"))
+        val racingGame = RacingGame(decider, cars, RaceBoard())
```

Comment:
만약 `fun interface`를 가장 마지막 파라미터로 받게된다면 
decider 객체를 직접 생성할 필요 없이, 아래와 같이 람다식으로 대체할 수 있어요. 😃

```suggestion
        val racingGame = RacingGame(cars, RaceBoard()) { true }
```

Reply:
네! 이런방법도 있었네요 😊 바로 적용해보도록 할게요!👍